www/jxcore/app.js

```diff
@@ -42,7 +44,18 @@ app.use(function allowCrossDomain(req, res, next) {
 });
 
 app.get('/', function (req, res) {
-  res.render('ejs/index',  { user: 'user' + Math.floor(Math.random() * 100) });
+  var cryptomanager = require('./thalicryptomanager');
+  cryptomanager.getPublicKeyHash(function (publicKeyHash) {
+    if (publicKeyHash == null) {
+      console.log('could not get the device publicKeyHash');
+      return;
+    } else {
+      console.log('got publicKeyHash');
+      var currentAddrEntry = addressPrefix + publicKeyHash;
```

Comment:
Please try to avoid abbreviations where possible, just spell out currentAddressEntry. We have auto-complete. It's o.k.